gulpfile.js

```diff
@@ -391,30 +413,22 @@ function release_win(arch) {
 
 // Create distribution package (zip) for windows and linux platforms
 function release(arch) {
-    var src = path.join(appsDir, pkg.name, arch);
-    var output = fs.createWriteStream(path.join(releaseDir, get_release_filename(arch, 'zip')));
-    var archive = archiver('zip', {
-        zlib: { level: 9 }
-    });
-    archive.on('warning', function (err) { throw err; });
-    archive.on('error', function (err) { throw err; });
-    archive.pipe(output);
-    archive.directory(src, 'Betaflight Configurator');
-    return archive.finalize();
+    var src = path.join(appsDir, pkg.name, arch, '**');
+    var output = get_release_filename(arch, 'zip');
+
+    return gulp.src(src)
+        .pipe(archiver(output, { zlib: { level: 9 } }))
+        .pipe(gulp.dest(releaseDir))
 }
 
 // Create distribution package for chromeos platform
 function release_chromeos() {
-    var src = distDir;
-    var output = fs.createWriteStream(path.join(releaseDir, get_release_filename('chromeos', 'zip')));
-    var archive = archiver('zip', {
-        zlib: { level: 9 }
-    });
-    archive.on('warning', function (err) { throw err; });
-    archive.on('error', function (err) { throw err; });
-    archive.pipe(output);
-    archive.directory(src, false);
-    return archive.finalize();
+    var src = path.join(distDir, '**');
+    var output = get_release_filename('chromeos', 'zip');
+
+    return gulp.src(src)
```

Comment:
In my test I've observed that in this way, we add the folder to the zip. So we must specify a `base` parameter, for example:
> return gulp.src(src, { base: distDir })

Reply:
Same comment than before.